addons/mail/static/src/discuss/call/common/call_action_list.js

```diff
@@ -113,4 +114,12 @@ export class CallActionList extends Component {
     async onClickToggleAudioCall(ev) {
         await this.rtc.toggleCall(this.props.thread);
     }
+
+    onTouchstartPushToTalk() {
+        this.rtc.onPushToTalk();
+    }
+
+    onTouchendPushToTalk() {
+        this.rtc.setPttReleaseTimeout();
+    }
```

Comment:
Having to name methods and make an indirection just for a single LOC that could be in the template itself, honestly not worth being them being methods. They could just be dump in `t-on-` values.